source_docs/installation/install-command-line.md

```diff
@@ -3,7 +3,9 @@ Quick-install, using the command-line
 
 If you have command-line access, you can easily install Bolt by executing a few
 commands. First, create the directory where you want to install Bolt, if it
-doesn't already exist. Enter the directory, and execute the following commands:
+doesn't already exist. We recommend installing Bolt outside the webroot, you can read more on this **here**. 
```

Comment:
I know this is kind of pervasive in the docs… but "webroot" is not valid in English, it would be "web root" or if you're really wanting to be hipster, "web-root"